src/iplant_groups/amqp.clj

```diff
@@ -31,3 +35,26 @@
     (declare-exchange channel exchange-cfg)
     (declare-queue channel exchange-cfg queue-cfg (keys handlers))
     (lc/blocking-subscribe channel (:name queue-cfg) (partial message-router handlers))))
+
+(defn publish-msg
```

Comment:
This function is basically just copied from other clojure services. I double-checked it looked right and I think it is, anyway.